.github/workflows/pr.yml

```diff
@@ -71,15 +71,16 @@ jobs:
           sh ./install-misspell.sh
           ./bin/misspell -error ./**/*
 
-  accept-pr:
+  required-status-check:
```

Comment:
FWIW this PR doesn't seem to require a change to this name, though also don't have a strong preference for either, am used to the current name now though.

Reply:
ya, this is a good point, I changed it back for now at least